benchmark/profile_restful_api.py

```diff
@@ -109,7 +73,8 @@ def read_dataset(tokenizer_path: str, dataset_path: str, samples: int,
         dataset = [data for data in dataset if len(data['conversations']) >= 2]
         # Only keep the first two turns of each conversation.
         dataset = [(data['conversations'][0]['value'],
-                    data['conversations'][1]['value']) for data in dataset]
+                    data['conversations'][1]['value'])
```

Comment:
It is not recommended. Let's keep it the same with vLLM.
https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_throughput.py